charts/trino/values.yaml

```diff
@@ -1053,3 +1055,30 @@ networkPolicy:
   #        port: 9999
   # ```
   egress: []
+
+additionalSecrets: false
+
+secrets: []
```

Comment:
Instead of having a generic list of secrets, how about adding specific secrets? It would be easier to document for users how to set it up. Also, the comment with the example below will not show up in the README, see other existing properties on how to format it correctly.